### DISCUSSION TITLE
pending state error when canceling the QR code scan.

### DISCUSSION THREAD

**Discussion by kdevkr** [MasterKale/SimpleWebAuthn#593] [Troubleshooting]
No error will occur if click '**Use passkey option on another device**' and cancel  '**Scan QR Code**' dialog.

Then, the following **startAuthentication** causes a pending state error.

> This error does not occur in PC browser.

I think it happens when you use the **autofill UI** and **login button** like https://www.passkeys.io/

> When cancel autofill promise using AbortController , but there is no error.

Does anyone know what caused this. device's Galaxy S23 and Chrome.

**Comment by MasterKale** [comment]
Hello @kdevkr, are you describing an error while trying to complete hybrid registration or authentication on a desktop/laptop running Chrome, and using your Galaxy S23 to scan the QR code that appears?

If so then knowing what version of which OS your desktop/laptop is running, what version of Chrome you're using, and what version of Android on your Galaxy smartphone might help get you an answer to your question.

**Reply by mambo-EIP** [reply]
Thank you for comment :)

**Application ver**
Chrome 126.0.6478.122

**Android ver**
Android 14; SM-S916N Build/UP1A.231005.007

I tested it more, it seems like a error from **Google Password Manager**.

Only occur from QR code scan prompt in Google Password Manager on Android chrome.

**Reply by MasterKale** [reply]
I'm sorry to hear you're experiencing this @mambo-EIP, but at least it sounds like you've narrowed it down to Google Password Manager. I'm not sure where to report bugs like this with GPM directly. I can try to find out for you unless you figure it out first.

**Reply by mambo-EIP** [reply]
I've solved the problem. 😅

It was caused by the direct use of the WebAuthn API for the conditional UI. After changing to use startAuthentication, I confirmed that waiting ceremony was canceled.

I think it's because I used AbortController without relying on the WebAuthnAbortService.

Thank you.
